packages/fluentui/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28136&commitId=348d5561da7625390cd5b3c38b19ea7be7f1b88e&env=prod)

✅ There was 0 screenshots added, 1 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.Chat Example Details Teams High Contrast.chromium.png|0| Removed|

</details>


<div id="vrtCommentFluentUI-v0"/>